fh-android-sdk/src/main/java/com/feedhenry/sdk/FH.java

```diff
@@ -418,10 +425,48 @@ public static int getLogLevel() {
 
     /**
      * Get the customized user-agent string for the SDK
-     * 
+     *
      * @return customized user-agent string
      */
     public static String getUserAgent() {
         return Device.getUserAgent();
     }
+
+    /**
+     *
+     * Registers a device to a push network.
+     *
+     * The push informations will be loaded from assets/push-config.json
+     *
+     * <pre>
+     * {
+     *   "pushServerURL": "<pushServerURL e.g http(s)//host:port/context >",
+     *   "android": {
+     *     "senderID": "<senderID e.g Google Project ID only for android>",
+     *     "variantID": "<variantID e.g. 1234456-234320>",
+     *     "variantSecret": "<variantSecret e.g. 1234456-234320>"
+     * }
+     * </pre>
+     *
+     * This method need to be called <b>after</b> a success {@link #init(android.content.Context, FHActCallback)}
+     *
+     * @param pContext  your application's context
+     * @param pCallback the pCallback function to be executed after the device registration is finished
+     */
+    public void pushRegister(final Context pContext, final FHActCallback pCallback) {
+        RegistrarManager.config("fh", AeroGearGCMPushJsonConfiguration.class)
+                .loadConfigJson(pContext)
+                .asRegistrar()
+                .register(pContext, new Callback<Void>() {
+                    @Override
+                    public void onSuccess(Void aVoid) {
+                        pCallback.success(new FHResponse(null, null, null, null));
```

Comment:
@wei-lee @danielpassos would it make sense to have at least a JIRA to give the `FHResponse` a slightly better improved ctor for cases like this, where you just pass in a lot of `null`s ?